examples/multiple-nodepools/README.md

```diff
@@ -0,0 +1,31 @@
+# Microsoft Azure Kubernetes Engine - Multiple Node Pools
+
+aks-engine supports creating a Kubernetes cluster with more than one node pool. These pools can have different configurations, such as VM size or storage profile.
+
+A cluster with multiple node pools can help you schedule CPU-intensive jobs to VM nodes with high processing power, or I/O intensive jobs to VMs with the fastest storage. Use [nodeSelectors][] or [resource requests][] to ensure that Pods are scheduled to nodes in the appropriate pool.
+
+A complete example is contained in the `multipool.json` apimodel in this directory. To add a node pool to an existing apimodel, just add another `agentPoolProfile` section:
```

Comment:
another **entry** in the `agentPoolProfile` section ?

Reply:
Done.